edb/schema/referencing.py

```diff
@@ -124,6 +140,7 @@ def derive_ref(
             old_bases = existing.get_bases(schema)
 
             if new_bases != old_bases:
+                assert isinstance(new_bases, so.ObjectList)
```

Comment:
This doesn't seem to work. Even if I move the declaration of new_bases, even if I specify its type at line 114; I still need this assertion to make mypy happy.

Otherwise it says: _"object" has no attribute "objects"_

Reply:
OK, fine then.